src/rest-server/src/models/v2/virtual-cluster/k8s.js

```diff
@@ -15,3 +15,23 @@
 // DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 // OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
+// module dependencies
+const vcConfig = require('@pai/config/vc');
+
+
+class VirtualCluster {
```

Comment:
please use functions directly

Reply:
It's doomed of a class, I think it's no need to unwrapper to functions.